src/components/Messages/Messages.jsx

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from "react-redux";
+import { set } from "../../slices/MessagesSlice";
+import store from "../../store";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import "./messages.scss";
+import "./ConsoleInterceptor.js";
+
+// A component that displays error messages
+function Messages() {
+  const messages = useSelector(state => state.messages);
+  return (
+    <div className="messages-pane">
+      {messages.items.map((message, index) => {
+        return (
+          <Alert icon={false} key={index} severity={message.severity}>
```

Comment:
What are your thoughts on the `SnackBar` component? (if we want to auto remove items)

Reply:
OR if we keep the alert. it should be allowed to be closed. 
additionally. can  you post a screenshot of what it looks like?

Reply:
SnackBar looks nice! I think it can help achieve the same level of functionnalities with way less code.

Were is the picture:
![image](https://user-images.githubusercontent.com/11857992/131301808-1508cd5c-7f2b-4dec-ac65-6a9f9eb549bf.png)

BTW I removed the icons because something is wrong in the CSS that make them ugly.

Reply:
I integrated SnackBar